jupyterhub/roles.py

```diff
@@ -239,26 +239,6 @@ def _check_scopes(*args, rolename=None):
                 )
 
 
-def _overwrite_role(role, role_dict):
-    """Overwrites role's description and/or scopes with role_dict if role not 'admin'"""
-    for attr in role_dict.keys():
-        if attr == 'description' or attr == 'scopes':
-            if role.name == 'admin':
```

Comment:
I moved the admin check to `create_role` so it would behave the same whether `admin` were in the db or not, which was not previously the case. Mostly a minor implementation detail.